index.html

```diff
@@ -144,10 +144,29 @@ <h2>Terminology</h2>
 <section id="object-spec">
     <h2>Object Specification</h2>
     <p>
-        An OCFL Object is a group of one or more content bitstreams (data and metadata), and their administrative
-        information that are together identified by a URI. The object may contain a sequence of versions of the
-        bitstreams that represent the evolution of the object's contents.
+        An OCFL Object is a group of one or more files, directories, and administrative
```

Comment:
I think there is a problem including "directories" as a fundamental component of an OCFL object because OCFL doesn't actually store directories (e.g. empty dir will be ignored). It does support "files organized in directories" so maybe that could be used as a replacement?

Reply:
How about instead:

> An OCFL Object is a group of one or more files, administrative information, and the directories they are organized in, that are together identified by a URI.

>